.github/labeler.yml

```diff
@@ -0,0 +1,12 @@
+# Rules defined here: https://github.com/actions/labeler
+type/chore:
+  - ./*.md
+  - ./**/*.yml
+  - ./acceptance/**/*
```

Comment:
Are the labels checked based on the order they are defined in this file? If so, wouldn't a **feature** with an added acceptance test or new dependency by marked as `type/chore` as oppose to `type/feature`?

Reply:
So what I think it does is it would add both labels. For example, that's what it did here: https://github.com/dfreilich/pack/pull/13. I opted to just make the rules simpler (so we at least have labels), and then we can adjust them as needed.